### PR TITLE
Bump ESLint version to add prefer-at condition

### PR DESCRIPTION
**Pull request by ShridharGoel**
### Details
Bump ESLint version to add prefer-at condition.

### Fixed Issues

$ https://github.com/Expensify/App/issues/43055
PROPOSAL: https://github.com/Expensify/App/issues/43055#issuecomment-2148253314
 
### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

</details>

<details>
<summary>MacOS: Desktop</summary>

</details>

### CODE REVIEW

**Review by fabioh8010 — COMMENTED** [review]
Did a partial review since there are actually hundreds of TS errors with the change to `.at()`

@ShridharGoel Whats the status of this PR? I think you can work on fixing the TS errors that are not related to [this PR](https://github.com/Expensify/eslint-config-expensify/pull/114) while we wait for that one. Please let's speed up the work here.

File: src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx [hunk 1]
```diff
@@ -196,7 +196,7 @@ function MoneyRequestPreviewContent({
                 a.type === CONST.VIOLATION_TYPES.VIOLATION ? -1 : 0,
             );
             if (violations?.[0]) {
```
[the remark this answers is not in the document]
Comment (fabioh8010) [reply src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx:198]: Ok, let's skip this improvement for now then.

**Comment by fabioh8010** [comment]
@ShridharGoel The ESLint PR was merged, could you prioritize work on this one? Thanks 🙏

File: src/libs/Navigation/linkingConfig/getMatchingCentralPaneRouteForState.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ function getMatchingCentralPaneRouteForState(state: State<RootStackParamList>, r
         return;
     }
 
-    const centralPaneName = TAB_TO_CENTRAL_PANE_MAPPING[topmostBottomTabRoute.name][0];
+    const centralPaneName = TAB_TO_CENTRAL_PANE_MAPPING[topmostBottomTabRoute.name].at(0);
```
Comment (fabioh8010) [inline src/libs/Navigation/linkingConfig/getMatchingCentralPaneRouteForState.ts:56]:
```suggestion
    const centralPaneName = TAB_TO_CENTRAL_PANE_MAPPING[topmostBottomTabRoute.name].at(0);
    if (!centralPaneName) {
        return;
    }
```

File: src/libs/Navigation/linkingConfig/getMatchingCentralPaneRouteForState.ts [hunk 3]
```diff
@@ -62,10 +62,10 @@ function getMatchingCentralPaneRouteForState(state: State<RootStackParamList>, r
     }
 
     if (topmostBottomTabRoute.name === SCREENS.HOME) {
-        return {name: centralPaneName, params: {reportID: getTopMostReportIDFromRHP(state)}};
+        return {name: centralPaneName ?? ({} as keyof CentralPaneScreensParamList), params: {reportID: getTopMostReportIDFromRHP(state)}};
```
Comment (fabioh8010) [inline src/libs/Navigation/linkingConfig/getMatchingCentralPaneRouteForState.ts:65]:
```suggestion
        return {name: centralPaneName, params: {reportID: getTopMostReportIDFromRHP(state)}};
```

File: src/libs/Navigation/linkingConfig/getMatchingCentralPaneRouteForState.ts [hunk 4]
```diff
@@ -62,10 +62,10 @@ function getMatchingCentralPaneRouteForState(state: State<RootStackParamList>, r
...
     }
 
-    return {name: centralPaneName};
+    return {name: centralPaneName ?? ({} as keyof CentralPaneScreensParamList)};
```
Comment (fabioh8010) [inline src/libs/Navigation/linkingConfig/getMatchingCentralPaneRouteForState.ts:68]:
```suggestion
    return {name: centralPaneName};
```

File: src/libs/Network/SequentialQueue.ts [hunk 5]
```diff
@@ -79,7 +79,11 @@ function process(): Promise<void> {
         return Promise.resolve();
     }
 
-    const requestToProcess = persistedRequests[0];
+    const requestToProcess = persistedRequests.at(0);
+
```
Comment (fabioh8010) [inline src/libs/Network/SequentialQueue.ts:83]:
```suggestion
```

File: src/libs/OptionsListUtils.ts [hunk 6]
```diff
@@ -295,7 +295,12 @@ Onyx.connect({
 
         // Iterate over the report actions to build the sorted and lastVisible report actions objects
         Object.entries(allReportActions).forEach((reportActions) => {
-            const reportID = reportActions[0].split('_')[1];
+            const reportID = reportActions[0].split('_').at(1);
+
```
Comment (fabioh8010) [inline src/libs/OptionsListUtils.ts:299]:
```suggestion
```

File: src/libs/OptionsListUtils.ts [hunk 7]
```diff
@@ -308,7 +313,13 @@ Onyx.connect({
                 sortedReportActions = ReportActionUtils.getCombinedReportActions(sortedReportActions, transactionThreadReportID, transactionThreadReportActionsArray, reportID);
             }
 
-            lastReportActions[reportID] = sortedReportActions[0];
+            const firstReportAction = sortedReportActions.at(0);
+
```
Comment (fabioh8010) [inline src/libs/OptionsListUtils.ts:317]:
```suggestion
```

File: src/libs/ReportActionsUtils.ts [hunk 8]
```diff
@@ -714,7 +714,9 @@ function replaceBaseURLInPolicyChangeLogAction(reportAction: ReportAction): Repo
         return updatedReportAction;
     }
 
+    // eslint-disable-next-line rulesdir/prefer-at
     if (Array.isArray(updatedReportAction.message) && updatedReportAction.message[0]) {
+        // eslint-disable-next-line rulesdir/prefer-at
```
Comment (fabioh8010) [inline src/libs/ReportActionsUtils.ts:719]: Same, why these suppressions?

File: src/libs/ReportActionsUtils.ts [hunk 9]
```diff
@@ -832,7 +834,7 @@ function getFirstVisibleReportActionID(sortedReportActions: ReportAction[] = [],
         return '';
     }
     const sortedFilterReportActions = sortedReportActions.filter((action) => !isDeletedAction(action) || (action?.childVisibleActionCount ?? 0) > 0 || isOffline);
-    return sortedFilterReportActions.length > 1 ? sortedFilterReportActions[sortedFilterReportActions.length - 2].reportActionID : '';
+    return sortedFilterReportActions.length > 1 ? sortedFilterReportActions.at(sortedFilterReportActions.length - 2)?.reportActionID ?? '' : '';
```
Comment (fabioh8010) [inline src/libs/ReportActionsUtils.ts:837]:
```suggestion
    return sortedFilterReportActions.length > 1 ? sortedFilterReportActions.at(sortedFilterReportActions.length - 2)?.reportActionID ?? '-1' : '';
```

File: src/libs/ReportUtils.ts [hunk 10]
```diff
@@ -4185,7 +4185,9 @@ function buildOptimisticTaskCommentReportAction(
     createdOffset = 0,
 ): OptimisticReportAction {
     const reportAction = buildOptimisticAddCommentReportAction(text, undefined, undefined, createdOffset, undefined, taskReportID);
-    if (Array.isArray(reportAction.reportAction.message) && reportAction.reportAction.message?.[0]) {
+    // eslint-disable-next-line rulesdir/prefer-at
+    if (Array.isArray(reportAction.reportAction.message) && reportAction.reportAction.message[0]) {
+        // eslint-disable-next-line rulesdir/prefer-at
```
Comment (fabioh8010) [inline src/libs/ReportUtils.ts:4190]: Same

File: src/libs/actions/IOU.ts [hunk 11]
```diff
@@ -3823,7 +3823,7 @@ function trackExpense(
 
 function getOrCreateOptimisticSplitChatReport(existingSplitChatReportID: string, participants: Participant[], participantAccountIDs: number[], currentUserAccountID: number) {
     // The existing chat report could be passed as reportID or exist on the sole "participant" (in this case a report option)
-    const existingChatReportID = existingSplitChatReportID || participants[0].reportID;
+    const existingChatReportID = existingSplitChatReportID || (participants.at(0)?.reportID ?? '');
```
Comment (fabioh8010) [inline src/libs/actions/IOU.ts:3826]:
```suggestion
    const existingChatReportID = existingSplitChatReportID || (participants.at(0)?.reportID ?? '-1');
```

File: src/libs/actions/IOU.ts [hunk 12]
```diff
@@ -5572,8 +5572,11 @@ function prepareToCleanUpMoneyRequest(transactionID: string, reportAction: OnyxT
     });
 
     if (ReportActionsUtils.getReportActionMessage(updatedReportPreviewAction)) {
-        if (Array.isArray(updatedReportPreviewAction?.message) && updatedReportPreviewAction.message?.[0]) {
+        // eslint-disable-next-line rulesdir/prefer-at
+        if (Array.isArray(updatedReportPreviewAction?.message) && updatedReportPreviewAction.message[0]) {
+            // eslint-disable-next-line rulesdir/prefer-at
             updatedReportPreviewAction.message[0].text = messageText;
+            // eslint-disable-next-line rulesdir/prefer-at
```
Comment (fabioh8010) [inline src/libs/actions/IOU.ts:5579]: Same

File: src/libs/actions/IOU.ts [hunk 13]
```diff
@@ -7925,9 +7928,11 @@ function mergeDuplicates(params: TransactionMergeParams) {
                     deleted: deletedTime,
                 },
                 ...(Array.isArray(reportAction.message) &&
+                    // eslint-disable-next-line rulesdir/prefer-at
                     !!reportAction.message[0] && {
                         message: [
                             {
+                                // eslint-disable-next-line rulesdir/prefer-at
```
Comment (fabioh8010) [inline src/libs/actions/IOU.ts:7935]: Same

File: src/libs/actions/Policy/Tag.ts [hunk 14]
```diff
@@ -426,6 +451,7 @@ function renamePolicyTag(policyID: string, policyTag: {oldName: string; newName:
                                 ...tag,
                                 name: newTagName,
                                 pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                                // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```
Comment (fabioh8010) [inline src/libs/actions/Policy/Tag.ts:454]: IS possible to remove these suppresions?

File: src/libs/actions/Report.ts [hunk 15]
```diff
@@ -852,7 +852,9 @@ function openReport(
     if (isCreatingNewReport) {
         // Change the method to set for new reports because it doesn't exist yet, is faster,
         // and we need the data to be available when we navigate to the chat page
+        // eslint-disable-next-line rulesdir/prefer-at
         optimisticData[0].onyxMethod = Onyx.METHOD.SET;
+        // eslint-disable-next-line rulesdir/prefer-at
```
Comment (fabioh8010) [inline src/libs/actions/Report.ts:857]: Same, why these suppresions?

File: src/libs/memoize/cache/ArrayCache.ts [hunk 16]
```diff
@@ -16,6 +16,7 @@ function ArrayCache<K, V>(config: CacheConfig<K>): Cache<K, V> {
      */
     function getKeyIndex(key: K): number {
         for (let i = cache.length - 1; i >= 0; i--) {
+            // eslint-disable-next-line rulesdir/prefer-at
```
Comment (fabioh8010) [inline src/libs/memoize/cache/ArrayCache.ts:19]: Same

File: src/pages/home/report/ReportActionItemSingle.tsx [hunk 17]
```diff
@@ -146,7 +147,7 @@ function ReportActionItemSingle({
         const avatarIconIndex = report.isOwnPolicyExpenseChat || ReportUtils.isPolicyExpenseChat(report) ? 0 : 1;
         const reportIcons = ReportUtils.getIcons(report, {});
 
-        secondaryAvatar = reportIcons[avatarIconIndex];
+        secondaryAvatar = reportIcons.at(avatarIconIndex) ?? getDefaultIcon;
```
Comment (fabioh8010) [inline src/pages/home/report/ReportActionItemSingle.tsx:150]:
```suggestion
        secondaryAvatar = reportIcons.at(avatarIconIndex) ?? fallbackIcon;
```

File: src/pages/home/report/ReportActionsView.tsx [hunk 18]
```diff
@@ -394,15 +396,15 @@ function ReportActionsView({
...
+                    newestReportAction?.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE)
             ) {
                 return;
             }
 
             didLoadNewerChats.current = true;
 
             if ((reportActionID && indexOfLinkedAction > -1) || !reportActionID) {
-                handleReportActionPagination({firstReportActionID: newestReportAction?.reportActionID});
+                handleReportActionPagination({firstReportActionID: newestReportAction?.reportActionID ?? ''});
```
Comment (fabioh8010) [inline src/pages/home/report/ReportActionsView.tsx:407]:
```suggestion
                handleReportActionPagination({firstReportActionID: newestReportAction?.reportActionID ?? '-1'});
```

File: src/pages/settings/Wallet/PaymentMethodList.tsx [hunk 19]
```diff
@@ -233,8 +233,10 @@ function PaymentMethodList({
                 // The card should be grouped to a specific domain and such domain already exists in a assignedCardsGrouped
                 if (assignedCardsGrouped.some((item) => item.isGroupedCardDomain && item.description === card.domainName) && !isAdminIssuedVirtualCard) {
                     const domainGroupIndex = assignedCardsGrouped.findIndex((item) => item.isGroupedCardDomain && item.description === card.domainName);
-                    assignedCardsGrouped[domainGroupIndex].errors = {...assignedCardsGrouped[domainGroupIndex].errors, ...card.errors};
+                    // eslint-disable-next-line rulesdir/prefer-at
+                    assignedCardsGrouped[domainGroupIndex].errors = {...assignedCardsGrouped.at(domainGroupIndex)?.errors, ...card.errors};
                     if (card.fraud === CONST.EXPENSIFY_CARD.FRAUD_TYPES.DOMAIN || card.fraud === CONST.EXPENSIFY_CARD.FRAUD_TYPES.INDIVIDUAL) {
+                        // eslint-disable-next-line rulesdir/prefer-at
                         assignedCardsGrouped[domainGroupIndex].brickRoadIndicator = CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR;
```
Comment (fabioh8010) [inline src/pages/settings/Wallet/PaymentMethodList.tsx:240]: Same

File: src/stories/PromotedActionBar.stories.tsx [hunk 20]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable rulesdir/prefer-at */
```
Comment (fabioh8010) [inline src/stories/PromotedActionBar.stories.tsx:1]: Why?

File: src/styles/utils/index.ts [hunk 21]
```diff
@@ -272,7 +272,7 @@ function getAvatarBorderStyle(size: AvatarSizeName, type: string): ViewStyle {
  */
 function getDefaultWorkspaceAvatarColor(text: string): ViewStyle {
     const colorHash = UserUtils.hashText(text.trim(), workspaceColorOptions.length);
-
+    // eslint-disable-next-line rulesdir/prefer-at
```
Comment (fabioh8010) [inline src/styles/utils/index.ts:275]: Why?

File: tests/actions/ReportTest.ts [hunk 22]
```diff
@@ -1,4 +1,6 @@
 /* eslint-disable @typescript-eslint/naming-convention */
+
+/* eslint-disable rulesdir/prefer-at */
```
Comment (fabioh8010) [inline tests/actions/ReportTest.ts:3]: Why?

File: tests/e2e/measure/math.ts [hunk 23]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable rulesdir/prefer-at */
```
Comment (fabioh8010) [inline tests/e2e/measure/math.ts:1]: Why?

File: tests/unit/APITest.ts [hunk 24]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable rulesdir/prefer-at */
```
Comment (fabioh8010) [inline tests/unit/APITest.ts:1]: Why?

File: tests/unit/MiddlewareTest.ts [hunk 25]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable rulesdir/prefer-at */
```
Comment (fabioh8010) [inline tests/unit/MiddlewareTest.ts:1]: Why?

File: tests/unit/WorkflowUtilsTest.ts [hunk 26]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable rulesdir/prefer-at */
```
Comment (fabioh8010) [inline tests/unit/WorkflowUtilsTest.ts:2]: Why?

**Review by fabioh8010 — COMMENTED** [review]
@ShridharGoel Thanks for the work on this. Some notes though:

- Please run `npm run gh-actions-build` every time you change any files under `.github/actions/javascript/` as we need to re-generate the compiled JS files.
- I'm seeing many ESLint suppresions, mainly `eslint-disable rulesdir/prefer-at` which defeats the purpose of this PR. Let's address and remove all them as much as possible.
- Let's remove `[NoQA]` from this PR's title as I think we should have extensive testing here due to changes in logic. @roryabraham WDYT about asking for a full regression test here?
- Once you solve the lint / workflow errors I think we can open the PR to review.

**Comment by roryabraham** [comment]
> WDYT about asking for a full regression test here?

We can do that, but let's do it on one platform only to save time and money

**Comment by fabioh8010** [comment]
@ShridharGoel Any updates here?

**Comment by blazejkustra** [comment]
@ShridharGoel Any updates on this?

**Comment by roryabraham** [comment]
I think tests don't matter as much. If excluding them from this rule helps get this PR merged sooner, I'm all for it 👍

**Comment by fabioh8010** [comment]
@ShridharGoel When do we expect to have updates here?

**Comment by blazejkustra** [comment]
@ShridharGoel Any updates?

**Comment by blazejkustra** [comment]
> Will be updating completely in the next 1-2 days.
Sorry but you didn't respond any of Fabio's comments, almost all checks are failing now + this PR is one month old 😢 

Given these, can we plan for this PR to be **ready to merge** in 1-2 days?

**Comment by fabioh8010** [comment]
@ShridharGoel We got conflicts in the PR. Please address them together with the workflow failures (except `Changed files ESLint check`, let's ignore this one for now)

Screenshot 2024-09-17 at 15 44 37

**Comment by fabioh8010** [comment]
@ShridharGoel By looking at the [workflow result](https://github.com/Expensify/App/actions/runs/10904970936/job/30262694402?pr=47439), it seems to be only these ones:

- src/pages/Search/SearchTypeMenuNarrow.tsx
- src/components/DistanceRequest/DistanceRequestFooter.tsx
- src/pages/home/report/ReportTypingIndicator.tsx

**Comment by fabioh8010** [comment]
@ShridharGoel There is a section about your error (`Invalid nesting in program blocks or scopes`) in [REACT_COMPILER.md](https://github.com/Expensify/App/blob/main/contributingGuides/REACT_COMPILER.md)

**Comment by eh2077** [comment]
@ShridharGoel This issue has been reassigned, please check this https://github.com/Expensify/App/issues/43055#issuecomment-2363363294. So, I think this PR will be on hold.

**Pull request closed without merging**